rs/sns/cli/BUILD.bazel

```diff
@@ -72,23 +72,6 @@ rust_library(
     deps = DEPENDENCIES,
 )
 
-rust_library(
-    name = "cli--test_feature",
-    srcs = glob(
-        ["src/**/*.rs"],
-        exclude = [
-            "**/*tests.rs",
-            "main.rs",
-        ],
-    ),
-    aliases = ALIASES,
-    crate_features = ["test"],
-    crate_name = "ic_sns_cli",
-    proc_macro_deps = MACRO_DEPENDENCIES,
-    version = "1.0.0",
-    deps = DEPENDENCIES_WITH_TEST_FEATURES,
-)
```

Comment:
Do we still have a use case where we need a test version of the SNS ClI?

Reply:
@aterga will be able to respond to this when he's back.  I don't recall the use case.  I would like to remove all of the test_feature stuff, as it's becoming quite a hassle when breaking up canister dependencies (in another PR)

Reply:
Just looking at the code and seeing there 
there are no `[cfg(feature = "test")]  nor [cfg(not(feature = "test"))]`  macros under the cli directory

the PR seems safe

Reply:
The PR is safe in the context of the IC monorepo.

The test version of the binary that can be used externally would go away, and that could cause problems in other testing scenarios.  DEPENDENCIES_WITH_TEST_FEATURES means that the test version of the binary is compiled against libraries with test features.  There were some behavioral differences at one point that were relevant for testing.  I want to make sure we don't break NNS Dapp testing workflows that depended on that.

Reply:
I completely missed DEPENDENCIES_WITH_TEST_FEATURES